### ISSUE TITLE
CREATE ROUTE TO ADD COMPANY GOAL.

### ISSUE THREAD

**Issue by devteni** [zurichat/zc_plugin_company_goals#324]
◾Create a route to add a new company goal. 
◾Take into account the model and conventions used for the controllers.

**Comment by masterd2020** [comment]
I am working on this

**Comment by devteni** [comment]
Okay. Did Chairman assign it to you?

**Comment by masterd2020** [comment]
Okay, Am not actually handling the authentication part.

**Comment by adeyinkar** [comment]
Am I to work on the authentication?

**Comment by devteni** [comment]
> Am I to work on the authentication?





> Am I to work on the authentication?

**Comment by devteni** [comment]
> Am I to work on the authentication?

Yes. But as it stands, there really is no go-to access for auth.

**Comment by devteni** [comment]
I might assign you another.

**Comment by adeyinkar** [comment]
Okay then. Let's see

**Comment by masterd2020** [comment]
#324 

[live link](https://goals.zuri.chat/api/v1/goals)

Sample payload with a POST Request
{
  "title": "Toheeb",
  "description": "description",
  "monthlyGoal": "monthly goal",
  "quarterlyGoal": "quarterly goal",
  "biannualGoal": "biannualGoal",
  "annualGoal": "annual goal",
  "achieved": false,
  "createdBy": "2021-09-20"
}
